build.gradle.kts

```diff
@@ -50,17 +47,20 @@ dependencies {
   implementation("com.pauldijou:jwt-core_2.11:4.3.0")
   implementation("com.google.code.gson:gson:2.8.6")
 
-  implementation("software.amazon.awssdk:sns:2.13.9")
+  implementation("software.amazon.awssdk:sns:2.13.13")
 
   testAnnotationProcessor("org.projectlombok:lombok:1.18.8")
   testCompileOnly("org.projectlombok:lombok:1.18.8")
   testImplementation("org.springframework.security:spring-security-test")
-  testImplementation("org.springframework.boot:spring-boot-starter-webflux")
   testImplementation("com.tngtech.java:junit-dataprovider:1.13.1")
   testImplementation("net.javacrumbs.json-unit:json-unit-assertj:2.17.0")
   testImplementation("io.github.http-builder-ng:http-builder-ng-apache:1.0.4")
   testImplementation("com.github.tomakehurst:wiremock-standalone:2.26.3")
   testImplementation("org.testcontainers:localstack:1.13.0")
   testImplementation("org.awaitility:awaitility-kotlin:4.0.2")
+  testImplementation("io.jsonwebtoken:jjwt:0.9.1")
+
+  testImplementation("org.springframework.security.oauth:spring-security-oauth2:2.4.1.RELEASE")
```

Comment:
only converted production code at present, hence dependencies moved to test scope